input/tail.md

```diff
@@ -417,6 +417,13 @@ If you see this message:
 
 It means that `fluentd` does not have read permission for `/path/to/file`. Check your fluentd and target files permission.
 
+**Note**: When `td-agent` is launched by systemd, the user of the `td-agent` process is the `td-agent` user.
```

Comment:
_by default_ is needed?

the user of the `td-agent` process is the `td-agent` user _by default_.

Reply:
Clarified in e6d9693